workflow/src/test/kotlin/com/r3/corda/sdk/token/workflow/FlowHelpers.kt

```diff
@@ -26,7 +26,7 @@ fun <T : TokenType> StartedMockNode.issueTokens(
         issueTo: StartedMockNode,
         notary: StartedMockNode,
         amount: Amount<T>? = null,
-        anonymous: Boolean = true
+        anonymous: Boolean = false
```

Comment:
Try not to change default values; other tests rely on them to be set a certain way and may impact test states and performance if changed.